students/17th/mikyung/login.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <!--head-->
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>westagram</title>
+    <link href="styles/login.css" rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Cookie&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://www.instagram.com/static/images/ico/favicon.ico/36b3ee2d91ed.ico"
+      rel="shortcut icon"
+      type="image/x-icon"
+    />
+  </head>
+
+  <!--body-->
+  <body>
+    <div class="first_div">
+      <h1>Westagram</h1>
+      <input
+        type="text"
+        class="input_login"
+        name="userID"
+        id="userID"
+        placeholder="전화번호, 사용자 이름 또는 이메일"
+      />
+      <input
+        type="password"
+        class="input_login"
+        name="userPW"
+        id="userPW"
+        placeholder="비밀번호"
+      />
+      <a href="#"> <button id="btn_login" disabled>로그인</button></a>
+      <div class="or_box">
+        <span class="or_under">또는</span>
+      </div>
+      <div class="fb">
+        <a href="#" class="fb_a">
+          <img src="img/3.png" width="16px" />
+          <span class="facebook_login"> Facebook으로 로그인</span></a
+        >
```

Comment:
```suggestion
          <span class="facebook_login"> Facebook으로 로그인</span>
        </a>
```
하나의 태그가 서로 다른 줄에 작성되어 있네요